### ISSUE TITLE
Error: $ operator is invalid for atomic vectors

### ISSUE THREAD

**Issue by Fpadt** [rstudio/bookdown#1246]
# Error: $ operator is invalid for atomic vectors
see [stackoverflow](https://stackoverflow.com/a/68877224/2756412)

## Reproducible example:
1. Download the GitHub repository [bookdown-demo](https://github.com/rstudio/bookdown-demo) as a Zip file, then unzip it locally.
2. Use the following code in the _output.yml (note only the last line was added)
```
    bookdown::gitbook:  
      css: style.css  
      config:  
        toc:
          before: |
            <li><a href="./">A Minimal Book Example</a></li>
          after: |
            <li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
        edit: https://github.com/rstudio/bookdown-demo/edit/master/%s
        download: ["pdf", "epub"]
        search: yes
```
3. Build the Book
```
==> rmarkdown::render_site(encoding = 'UTF-8')

Error: $ operator is invalid for atomic vectors
Execution halted

Exited with status 1.
```
4. change _output.yml to teh following code
**note the last two lines**
```
bookdown::gitbook:
  css: style.css
  config:
    toc:
      before: |
        <li><a href="./">A Minimal Book Example</a></li>
      after: |
        <li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
    edit: https://github.com/rstudio/bookdown-demo/edit/master/%s
    download: ["pdf", "epub"]
    search:
      engine: fuse

```

5. Result:
```
output file: bookdown-demo.knit.md


Output created: _book/index.html
```

## sessionInfo()
R version 3.5.2 (2018-12-20)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                           LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] compiler_3.5.2   bookdown_0.23    fastmap_1.1.0    rsconnect_0.8.24 htmltools_0.5.2  tools_3.5.2     
 [7] yaml_2.2.1       rmarkdown_2.10   knitr_1.33       xfun_0.25        digest_0.6.27    packrat_0.7.0   
[13] rlang_0.4.11     evaluate_0.14

**Comment by yihui** [comment]
Duplicate of #1238

**Comment by Fpadt** [comment]
apologies

**Comment by yihui** [comment]
No worries! Next time please try to follow the issue guide to test the dev version first. Thank you!
